test/unittest/unittest_nntrainer_layers.cpp

```diff
@@ -2118,7 +2118,7 @@ TEST_F(nntrainer_AdditionLayer, setProperty_02_n) {
   EXPECT_EQ(status, ML_ERROR_INVALID_PARAMETER);
 }
 
-TEST_F(nntrainer_AdditionLayer, forwarding_01_n) {
+TEST_F(nntrainer_AdditionLayer, DISABLED_forwarding_01_n) {
```

Comment:
This was throwing because of the bug revealed in this patch.

setting `num_inputs=1` does not throw in addition layer forwarding.

Reply:
can you apply patch for this bug in this PR as well?

Reply:
I'd like to ask if it is a negative case or not, because

1. we don't usually verify input when forwarding
2. even in intialize() function, num_input = 1 is allowed.

Reply:
As far as I can see, this unit test throws due to input dimension mismatch and not because of num_input (there is no check on number of inputs in addition layer).
This can be removed addition layer as long as tensor class throws in this case when dimension mismatch.

Reply:
Clarified. I'll delete this test :)